pkg/kubefed2/federate/enable.go

```diff
@@ -168,6 +176,19 @@ func EnableFederation(config *rest.Config, federationNamespace, key, primitiveGr
 	glog.V(2).Infof("Found resource %q", resourceKey(*apiResource))
 
 	typeConfig := typeConfigForTarget(*apiResource, comparisonField, overridePaths, primitiveGroup, primitiveVersion)
+	concreteTypeConfig := typeConfig.(*fedv1a1.FederatedTypeConfig)
+
+	// TODO(marun) Retrieve the validation schema of the target and
+	// use it in constructing the schema for the template.
+	crds := primitiveCRDs(typeConfig)
+
+	if outputYAML {
+		objects := []pkgruntime.Object{concreteTypeConfig}
+		for _, crd := range crds {
+			objects = append(objects, crd)
+		}
+		writeObjectsToYAML(objects, cmdOut)
```

Comment:
Missed catching the returned error here ?

Reply:
Done.